### PR TITLE
Dropdown list routes with search box

### PR DESCRIPTION
**Pull request by JulienMrgrd**
Related to #99.

- The dropdown :

- The search box :
registrysearchgif

All "down" routes are not clickable.
You can search instances by app name with the search box (inside the dropdown).

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
so nice @JulienMrgrd !

**Comment by deepu105** [comment]
Good work. Ill review it soon

**Comment by JulienMrgrd** [comment]
Please don't merge for the moment.
I'm testing on Firefox (I use Chrome usually), and there are some CSS issues :angry:

File: src/main/webapp/content/scss/global.scss [hunk 1]
```diff
@@ -39,19 +39,10 @@ Generic styles
     border: none;
 }
 
-.error {
```
Comment (deepu105) [inline src/main/webapp/content/scss/global.scss]: why is this removed? its a common class for error

Comment (JulienMrgrd) [inline src/main/webapp/content/scss/global.scss]: I removed some unused css in this file, and I didn't see anything about this.
Should be reverted ?

Comment (deepu105) [inline src/main/webapp/content/scss/global.scss]: ok then. just fix the remaining comments

**Comment by JulienMrgrd** [comment]
@deepu105, with the latest Chrome update ([58.0.3029.81](https://chromereleases.googleblog.com/2017/04/stable-channel-update-for-desktop.html)), the value of the button (to select instances) cannot be updated.

giphy
_in route-selector.component.html :_
```html
<button class="btn btn-outline-primary" id="sortMenu" ngbDropdownToggle>
{{ activeRoute.serviceId ? (activeRoute.serviceId | uppercase) : (activeRoute.appName | uppercase)}}
</button>
```
Even if we changed the activeRoute value, the button value stay the same (but the real activeRoute change, because all new values, like metrics or others, are loaded).

I've already tried to not bind the button with a "Route" but with a simple string, or to change the Subscription "routeChanged$" call/callback, without success...
Do you have any idea ?

It's ok on Firefox.
